doc/versions.json

```diff
@@ -19,6 +20,12 @@
       "zephyr": "2.4.0",
       "mcuboot": "1.6.99"
     },
+    "1.4.1": {
+      "ncs": "1.4.1",
+      "nrfxlib": "1.4.0",
+      "zephyr": "2.4.0-ncs2",
```

Comment:
This is a tag, but not a documentation version.
It should either be 2.4.99 (if there are any changes) or 2.4.0.

@carlescufi , you discussed that with @joerchan for the v1.3.2 release and said the Zephyr version should stay as is, even if there's a new tag?

Reply:
Talked about this with @rlubos , and there are no changes that we're including from upstream except some cherry-picks. Will revert to 2.4.0.